Framework/DataHandling/src/LoadNXSPE.cpp

```diff
@@ -135,6 +135,11 @@ void LoadNXSPE::exec() {
     file.getData(temporary);
     psi = temporary.at(0);
     file.closeData();
+    if (!isfinite(psi)) {
```

Comment:
You should either use `std::isfinite` or `boost::math::isfinite` like elsewhere in the file. I would change both to `std::isfinite`